avocado/utils/multipath.py

```diff
@@ -67,6 +67,7 @@ def device_exists(mpath):
     """
     Checks if a given mpath exists.
 
+    :param mpath: The multipath path
```

Comment:
How about parameter type too? Since this is being fixed the extra step of adding the parameter types might be good.